libr/include/r_types.h

```diff
@@ -462,6 +462,17 @@ static inline void *r_new_copy(int size, void *data) {
 #ifndef eprintf
 #define eprintf(...) fprintf (stderr, __VA_ARGS__)
 
+/* These macros may be turned off at will. This is so that they can be left in
+ * the codebase while running tests. Disabling USE_DEBUG_VAR_PRINTS will
+ * prevent them from affecting program output so tests can be run as normal
+ * after recompiling.
+ *
+ * Note to maintainers: This macro should be left enabled in production, but
+ * the function macros should be unused. The functions are supposed to be
+ * immediately available to anyone hacking on the source.
+ */
+#define USE_DEBUG_VAR_PRINTS 1
```

Comment:
uhmf the user cant override this value with an env var or configure option.. so it seems like that should be wrapped inside an #ifdef